Survey/Survey/Logger.swift

```diff
@@ -6,12 +6,18 @@ class Logger {
     private let log: OSLog
 
     private init() {
-        self.log = OSLog(subsystem: Bundle.main.bundleIdentifier ?? "com.yourapp.default", category: "General")
+        self.log = OSLog(
+            subsystem: Bundle.main.bundleIdentifier ?? "Incredihire-LLC.Survey",
```

Comment:
There's a naming convention that needs to be followed for the bundle. it's going to be something like com.incredihire.survey (i.e. com.company.application)